pints/_mcmc/__init__.py

```diff
@@ -84,6 +84,9 @@ def __init__(self, x0, sigma0=None):
         # Get number of parameters
         self._n_parameters = len(self._x0)
 
+        # State of sampler
+        self._active = True
```

Comment:
I'd prefer not to have this in every optimiser - it's really only something the Controller needs to know about. So better just being a list inside `run()` I think

(Optimisers can also be used without a controller, for example!)